src/components/authentication/login.jsx

```diff
@@ -67,8 +69,10 @@ export class Login extends Component {
     const { isAuthanticated } = auth;
     if (isAuthanticated && token) {
       Helpers.setToken(token);
+      const userInfo = Helpers.getUserInfoFromToken();
+      localStorage.setItem('user', JSON.stringify(userInfo));
     }
-    isAuthanticated && this.props.history.replace('/articles');
+    isAuthanticated && window.location.replace('/articles');
```

Comment:
what is the reason for changing it?

Reply:
When a user logs in, the page was not being refreshed so that the token can be taken. As long as the app is not refreshed, the user remains unauthenticated. So we decided to refresh the app at login to get rid of the issue.